std_msgs/Header header

ChordophoneString[] strings

# optional vector indicating general approach direction for plucking (e.g., "up" for lying instruments)
geometry_msgs/Vector3 pluck_approach